### ISSUE TITLE
when inline is disabled, an error is thrown

### ISSUE THREAD

**Issue by brecert** [darionco/rollup-plugin-web-worker-loader#22]
when the inline setting is disabled the following error is throw.

idMap is not defined in [generateBundle.js](https://github.com/darionco/rollup-plugin-web-worker-loader/blob/master/src/plugin/generateBundle.js)

```
ReferenceError: idMap is not defined
    at generateBundle (C:\Users\-\code\html\see\node_modules\rollup-plugin-web-worker-loader\src\plugin\generateBundle.js:6:30)
    at Object.generateBundle (C:\Users\-\code\html\see\node_modules\rollup-plugin-web-worker-loader\src\index.js:61:13)
    at C:\Users\-\Code\html\see\node_modules\rollup\dist\shared\rollup.js:18100:25
```

Reverting back to 1.1.3 fixes this issue for now.

**Comment by thecreazy** [comment]
Same to me

actually, looking inside this [file](https://github.com/darionco/rollup-plugin-web-worker-loader/blob/master/src/plugin/generateBundle.js) the idMap variables is not defined


Edit 

Created a pull request with the fix here: https://github.com/darionco/rollup-plugin-web-worker-loader/pull/23

**Comment by darionco** [comment]
Fixed in 1.2.2
Thanks!
